libnmstate/nm/ipv4.py

```diff
@@ -33,6 +42,7 @@ def create_setting(config):
 
 
 def _add_addresses(setting_ipv4, addresses):
+    setting_ipv4.clear_addresses()
```

Comment:
Why it is done twice? (line 29 does it as well)

Reply:
Just some leftover. Fixed.